cmd/frontend/internal/app/debug_test.go

```diff
@@ -154,3 +157,79 @@ func TestGrafanaLicensing(t *testing.T) {
 		}
 	})
 }
+
+func TestSentryTunnel(t *testing.T) {
+	mockProjectID := "1334031"
+	var sentryPayload = []byte(fmt.Sprintf(`{"event_id":"6af2790372f046689a858b1d914fe0d5","sent_at":"2022-07-07T17:38:47.215Z","sdk":{"name":"sentry.javascript.browser","version":"6.19.7"},"dsn":"https://randomkey@o19358.ingest.sentry.io/%s"}
+{"type":"event","sample_rates":[{}]}
+{"message":"foopff","level":"info","event_id":"6af2790372f046689a858b1d914fe0d5","platform":"javascript","timestamp":1657215527.214,"environment":"production","sdk":{"integrations":["InboundFilters","FunctionToString","TryCatch","Breadcrumbs","GlobalHandlers","LinkedErrors","Dedupe","UserAgent"],"name":"sentry.javascript.browser","version":"6.19.7","packages":[{"name":"npm:@sentry/browser","version":"6.19.7"}]},"request":{"url":"https://sourcegraph.test:3443/search","headers":{"Referer":"https://sourcegraph.test:3443/search","User-Agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.53 Safari/537.36"}},"tags":{},"extra":{}}`, mockProjectID))
+
+	router := mux.NewRouter()
+	addSentry(router)
+
+	t.Run("POST /_sentry_tunnel", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("POST sentry tunnel", func(t *testing.T) {
```

(keeping it vague to avoid it becoming outdated like it did here 😛 )

Reply:
hahaha, good one :D